### PR TITLE
Update THIRD_PARTY_LICENSES_RUST_CRATES.html

### PR DESCRIPTION
**Pull request by raulmldv**
Fixed failing `nitro-about` target on recent PRs.

Signed-off-by: Raul-Ovidiu Moldovan <raulmldv@amazon.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by shtaked — APPROVED** [review]

**Comment by petreeftime** [comment]
I think we should simply remove nitro-about from the default test set, or keep just a check for valid licenses. We only need to include this file when we produce a binary distribution.

**Comment by raulmldv** [comment]
> I think we should simply remove nitro-about from the default test set, or keep just a check for valid licenses. We only need to include this file when we produce a binary distribution.

Couldn't find a quick check using `cargo-about` or `cargo-license`. I will remove `nitro-about` for now and look more into it. I think I can leave the licenses updated so I'll push here.

File: Makefile [hunk 1]
```diff
@@ -241,17 +241,6 @@ nitro-audit: build-setup build-container
 			'source /root/.cargo/env && \
 			cargo audit -f /nitro_src/Cargo.lock --ignore RUSTSEC-2020-0159 --ignore RUSTSEC-2020-0071'
 
-nitro-about: build-setup build-container
```
Comment (petreeftime) [inline Makefile:244]: Why are you removing this?

Comment (raulmldv) [reply Makefile:244]: Right, I can just remove it from the script.

Comment (petreeftime) [reply Makefile]: Let's merge #425 first.

**Pull request merged**